firebase-installations/src/main/java/com/google/firebase/installations/StateListener.java

```diff
@@ -18,4 +18,6 @@
 
 interface StateListener {
   boolean onStateReached(PersistedFidEntry persistedFidEntry);
+
+  boolean onException(PersistedFidEntry persistedFidEntry, Exception exception);
```

Comment:
might be good to document the meaning behind the boolean return value

Reply:
Updated. Thanks.